google/cloud/aiplatform/metadata/schema/base_execution.py

```diff
@@ -112,3 +114,72 @@ def create(
             credentials=credentials,
         )
         return self.execution
+
+    def start_execution(
+        self,
+        *,
+        metadata_store_id: Optional[str] = "default",
+        resume: bool = False,
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+    ) -> "execution.Execution":
+        """Create and starts a new Metadata Execution or resumes a previously created Execution.
+
+        To start a new execution:
+
+        ```
+        with execution_schema.ContainerExecution(display_name='preprocess').start_execution() as exc:
+          exc.assign_input_artifacts([my_artifact])
+          model = aiplatform.Artifact.create(uri='gs://my-uri', schema_title='system.Model')
+          exc.assign_output_artifacts([model])
+        ```
+
+        To continue a previously created execution:
+        ```
+        with execution_schema.ContainerExecution(resource_id='my-exc', resume=True) as exc:
+            ...
+        ```
+
+        Args:
+            metadata_store_id (str):
+                Optional. The <metadata_store_id> portion of the resource name with
+                the format:
+                projects/123/locations/us-central1/metadataStores/<metadata_store_id>/executions/<resource_id>
+                If not provided, the MetadataStore's ID will be set to "default". Currently only the 'default'
+                MetadataStore ID is supported.
+            resume (bool):
+                Resume an existing execution. If resume set to `True`, resource_id must be provided.
```

Comment:
Does this comment need to be updated? Seems like we don't require a resource_id arg in the function.